guides/v2.3/config-guide/elasticsearch/es-downgrade.md

```diff
@@ -0,0 +1,45 @@
+---
+group: configuration-guide
+title: Change Elasticsearch Client
+functional_areas:
+  - Configuration
+  - Search
+  - System
+  - Setup
+---
+
+The Magento 2.3.1 update adds support for Elasticsearch (ES) 6.x.
+Magento still provides connectivity for ES 2.x and 5.x but this will need to be specifically enabled.
+
+If you need to run Magento 2.3.1 with Elasticsearch 2.x or 5.x, you must change the Elasticsearch [PHP client][] version.
+
+Both ES 2.x and 5.x are [End of Life][].
+Running ES 2.x is strongly discouraged.
+
+## Change the Elasticsearch Client version
+
+If you need to work with Elasticsearch 5.x, run the following command:
+
+```bash
+composer require "elastic/elasticsearch:~5.1"
```

Comment:
We previously claimed support for 5.2. Check if this version can be bumped up to 5.2

Reply:
Good catch.